resolwe_bio/kb/tests/test_feature.py

```diff
@@ -192,6 +192,13 @@ def test_feature_search(self):
         )
         self.assertEqual(len(response.data), 1)
 
+    def test_serialization(self):
+        FEATURE_SEARCH_URL = reverse("resolwebio-api:kb_feature_search")
+
+        response = self.client.get(FEATURE_SEARCH_URL, {"limit": 1}, format="json")
+        self.assertEqual(len(response.data["results"]), 1)
+        self.assertTrue("id" in response.data["results"][0])
```

Comment:
It would be great to check all fields, not only `id`.

Reply:
Good point, thanks. Done.